speakerfight/settings.py

```diff
@@ -40,6 +40,7 @@
 
 STATIC_ROOT = os.path.join(BASE_DIR, "static")
 DEFAULT_FROM_EMAIL = NO_REPLY_EMAIL = 'Speakerfight <noreply@speakerfight.com>'
+FEEDBACK_EMAIL = 'Speakerfight <feedback@speakerfight.com>'
```

Comment:
Yes, it's possible. However, I think that the 'DEFAULT_FROM_EMAIL' setting has not the same goal as 'FEEDBACK_EMAIL'. Note that the former is commonly used to discourage the user to reply any contact, while the latter should encourage continuous communication. This is the rationale came with when developing the feature.